server/routes/createGoal/createGoalController.test.ts

```diff
@@ -0,0 +1,117 @@
+import { Request, Response, NextFunction } from 'express'
+import { SessionData } from 'express-session'
+import CreateGoalController from './createGoalController'
+import { PrisonerSearchService } from '../../services'
+import validateCreateGoalForm from './createGoalFormValidator'
+import validateAddStepForm from './addStepFormValidator'
+
+jest.mock('./addStepFormValidator')
+jest.mock('./createGoalFormValidator')
+
+describe('createGoalController', () => {
```

Comment:
I spotted we didn't have a test for the controller earlier and wondered if we were just going to rely on the int-tests. However, you've added more logic in it now, so it makes sense 👍